dags/hivemind_etl_helpers/src/db/github/extract/__init__.py

```diff
@@ -1,5 +1,6 @@
 # flake8: noqa
-from .comments import fetch_comments
 from .commit import fetch_commits
+from .github_comment_extraction import GitHubCommentExtraction
```

Comment:
As the `GitHubCommentExtraction` is not going to be accessed directly and we would always use `GitHubExtraction` to use the fetching of comments, commits, etc I suggest to remove this line and for the case of test cases this can be directly accessed from its file.